### ISSUE TITLE
Avoid SSE3 and SSSE3 dependency

### ISSUE THREAD

**Issue by torognes** [torognes/swarm#46]
The Swarm code is compiled with "-mssse3" in the Makefile. A few ssse3 instructions are used in the alignment code if present, otherwise the code is written to only require sse2, and the presence of sse2 is tested for. However, it seems that the compiler also uses some sse3 or ssse3 instructions somewhere. This results in termination of Swarm with "Illegal instruction" fatal errors on some architectures that does not support all sse3 or ssse3 instructions. Two reports from users indicate this (both using pick_denovo_otus.py in qiime).

The code should be recompiled with other compiler options in the Makefile to avoid this problem.

**Comment by colinbrislawn** [comment]
Qiime is configured to uses swarm 1.2.19 followed by swarm_breaker.py. If qiime users are reporting this, the problem may be replicable in an older version of swarm.

**Comment by frederic-mahe** [comment]
I've tested the new version 1.2.21 and it works correctly on my computer (clustering results are unchanged). I've tried to compare the behavior of versions 1.2.20 and 1.2.21 on the oldest cluster nodes I could find (PowerEdge M605 Blade Server from 2009, AMD Opteron), but they both behave similarly, without any error messages. The next step is to ask the QIIME maintainers to point to a new release of swarm to see if it solves the SSE3 dependency problem reported by a few users.
